website/source/docs/providers/azurerm/r/managed_image.html.markdown

```diff
@@ -0,0 +1,99 @@
+---
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_managed_image"
+sidebar_current: "docs-azurerm-resource-managed-image"
+description: |-
+  Create a Managed Image.
+---
+
+# azurerm\_managed\_image
+
+Create a managed image.
+
+## Example Usage with Create from VHD
+
+```hcl
+resource "azurerm_resource_group" "test" {
+  name = "acctest"
+  location = "West US"
+}
+
+resource "azurerm_managed_image" "test" {
+  name = "acctest"
+  location = "West US"
+  resource_group_name = "${azurerm_resource_group.test.name}"
+
+	os_disk {
+    os_type = "Linux"
+    os_state = "Generalized"
+		blob_uri = "{blob_uri}"
+    size_gb = 30
+	}
+}
+```
+
+## Example Usage with Create from Virtual Machine (VM must be generalized prior)
```

Comment:
"must be generalized beforehand" or "first"

(prior is an adjective, not an adverb, "prior to" is a preposition)